### DISCUSSION TITLE
[1593500]战神4 - God of War {本体}

### DISCUSSION THREAD

**Discussion by lltcom** [pjy612/SteamManifestCache#764] [Share]
### Steam 游戏ID

1593500

### 密钥列表

```hash
1593501;77f7390a115063bccc41240f71e0bf27d42c6a7c0424657756fe3f99f26a287a
```

### 清单附件

[[1593500]战神4 - God of War {本体}.zip](https://github.com/user-attachments/files/16615427/1593500.4.-.God.of.War.zip)

### 备注

lltcom@gmail.com

**Comment by barryhamsy** [comment]
Thank you!
